doc/source/math/ref_frame.rst

```diff
@@ -19,14 +19,14 @@ documentation:
     system.  kOS inherits this behavior from KSP.
 
 In all the reference frames mentioned below, the orientation of 
-the axes is **left-handed**.  This means that if you imagine opening your
+the axes is **left-handed**. This means that if you imagine opening your
 palm and pointing your fingers down the x-axis, then curling your fingers
 in the direction of the y-axis, then sticking your thumb up, that the
-direction your thumb would have to be pointing to accomplish this task
-is the direction of the z-axis **if and only if** you used your left
-hand to perform those steps.  (If you do those steps with your right
-hand, you get a z-axis in the opposite direction and that is known as
-a **right handed** coordinate system).
+direction in which your thumb must be pointing is the direction of the
```

Comment:
"direction in which your thumb **would** be pointing" instead of "**must** be pointing" might be a better way to say it, as it matches with how things are phrased up above.